modules/apps/forms-and-workflow/calendar/calendar-service/src/main/java/com/liferay/calendar/service/impl/CalendarBookingServiceImpl.java

```diff
@@ -427,13 +427,27 @@ public CalendarBooking invokeTransition(
 			calendarBookingId, instanceIndex);
 
 		return invokeTransition(
-			calendarBookingId, status, calendarBookingInstance.getStartTime(),
+			calendarBookingId, calendarBookingInstance.getStartTime(), status,
 			updateInstance, allFollowing, serviceContext);
 	}
 
+	/**
+	 * @deprecated As of 2.2.0, replaced by {@link #invokeTransition(long, int,
+	 *             long, boolean, boolean, ServiceContext)}
+	 */
+	@Deprecated
+	@Override
+	public void invokeTransition(
+			long calendarBookingId, int status, ServiceContext serviceContext)
+		throws PortalException {
+
+		invokeTransition(
+			calendarBookingId, status, 0, false, false, serviceContext);
+	}
+
 	@Override
 	public CalendarBooking invokeTransition(
-			long calendarBookingId, int status, long startTime,
+			long calendarBookingId, long startTime, int status,
 			boolean updateInstance, boolean allFollowing,
 			ServiceContext serviceContext)
 		throws PortalException {
```

Comment:
Hey guys! @mbowerman @natocesarrego

I just changed the order of the arguments as requested by Brian Chan in https://github.com/brianchandotcom/liferay-portal/pull/44044#issuecomment-254886443. This is an interesting part of our code standards.

Thanks!